nss-day-cohort-39/nutshell-smaragdine-sugar-gliders
# Navigation and Login Style

##Changes
-added a navigation bar to top of page
-styled the login container and buttons
-added a logout button that only renders once a user is logged in and clears when a user logs out

##To test
-NOTE: You must serve from the src directory for the images to render
-Test that everything renders on the log in page correctly
-Test that the logout button appears only once a user is logged in and disappears once a user is log
-NOTE: styling may be off for dashboard page
